### PR TITLE
Upstream Merge May 3rd

### PR DESCRIPTION
**Pull request by Jolly-66**
I don't remember see a config update for the new voting options, but them and a format error in uplink are causing errors this time

blah blah blah probs touched something i shouldn't have

image

### CODE REVIEW

**Comment by Jolly-66** [comment]
who will win

jolly pulling from the upstream

some lines of code that tell us the code isn't fucking stupid

**Comment by prodirus** [comment]
git checkout upstream/master _maps/RandomRuins/SpaceRuins/gasthelizards.dmm
git checkout upstream/master _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm

run these two lines then commit u fool

**Comment by Jolly-66** [comment]
> git checkout upstream/master _maps/RandomRuins/SpaceRuins/gasthelizards.dmm
> git checkout upstream/master _maps/RandomRuins/LavaRuins/lavaland_surface_dead_ratvar.dmm
> 
> run these two lines then commit u fool

i'll commit u

File: code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm [hunk 1]
```diff
@@ -270,38 +263,47 @@
 
 	. = TRUE
 	var/mob/living/carbon/carbon_user = user
-	for(var/obj/item/living_heart/LH in atoms)
+	for(var/obj/item/living_heart/heart in atoms)
 
-		if(LH.target && LH.target.stat != CONSCIOUS) // NON-MODULE CHANGE
+		if(heart.target && heart.target.stat == DEAD)
```
Comment (MrMelbert) [inline code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm:268]:
```suggestion
		if(heart.target && heart.target.stat != CONSCIOUS) // NON-MODULE CHANGE
```

File: code/modules/client/preferences_savefile.dm [hunk 2]
```diff
@@ -224,10 +224,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 			parsed_favs += path
 	favorite_outfits = uniqueList(parsed_favs)
 
-	// NON-MODULE CHANGES: client prefs
-	READ_FILE(S["hear_speech_sounds"] , hear_speech_sounds)
-	READ_FILE(S["hear_radio_sounds"] , hear_radio_sounds)
-
```
Comment (MrMelbert) [inline code/modules/client/preferences_savefile.dm:230]: deleted

File: code/modules/client/preferences_savefile.dm [hunk 3]
```diff
@@ -277,10 +273,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	key_bindings = sanitize_keybindings(key_bindings)
 	favorite_outfits = SANITIZE_LIST(favorite_outfits)
 
-	// NON-MODULE CHANGES: sanitization
-	hear_speech_sounds = sanitize_integer(hear_speech_sounds, FALSE, TRUE, initial(hear_speech_sounds))
-	hear_radio_sounds = sanitize_integer(hear_speech_sounds, FALSE, TRUE, initial(hear_radio_sounds))
-
```
Comment (MrMelbert) [inline code/modules/client/preferences_savefile.dm:283]: deleted

File: code/modules/client/preferences_savefile.dm [hunk 4]
```diff
@@ -358,11 +350,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	WRITE_FILE(S["key_bindings"], key_bindings)
 	WRITE_FILE(S["hearted_until"], (hearted_until > world.realtime ? hearted_until : null))
 	WRITE_FILE(S["favorite_outfits"], favorite_outfits)
-
-	// NON-MODULE CHANGES: client prefs
-	WRITE_FILE(S["hear_speech_sounds"] , hear_speech_sounds)
-	WRITE_FILE(S["hear_radio_sounds"] , hear_radio_sounds)
-
```
Comment (MrMelbert) [inline code/modules/client/preferences_savefile.dm:365]: deleted

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Pull request merged**